presto-main/src/main/java/com/facebook/presto/connector/system/QuerySystemTable.java

```diff
@@ -170,7 +170,7 @@ private static void appendToBlockBuilder(Type type, String value, BlockBuilder b
         }
     }
 
-    private static Long toMillis(Duration duration)
+    static Long toMillis(Duration duration)
```

Comment:
Instead of exporting this, I would simply replace the future usages with Optional.ofNullable(duration).map(Duration::toMillis).orElse(null)